cl/search/migrations/0037_add_harvard_id_to_opinioncluster.sql

```diff
@@ -0,0 +1,8 @@
+BEGIN;
+ALTER TABLE "search_opinioncluster" ADD COLUMN "harvard_id" varchar DEFAULT '0' NOT NULL;
+ALTER TABLE "search_opinioncluster" ALTER COLUMN "harvard_id" DROP DEFAULT;
```

Comment:
it would seem that this implicates #1106, but from the issue and the [migration page](https://github.com/freelawproject/courtlistener/wiki/Database-migrations) there’s not really any particular action to take? or it’s on a case-by-case basis?

Dunno - I welcome feedback on this.

Reply:
You're referring to the new column with a default? Happily that's fine these days.